### PR TITLE
Update url

### PR DESCRIPTION
**Pull request by jburel**
Replace http by https

cc @sbesson @chris-allan

### CODE REVIEW

**Comment by jburel** [comment]
Looks like some travis fun again on that repo!!

**Comment by sbesson** [comment]
The errors are legimate unit tests errors as HTTP URLs are used all over the place:

```
$ git grep "http://www.openmicroscopy.org" | wc
     139     569   15960
```

**Comment by jburel** [comment]
I did not check :-) I assume it was some travis fun after the recent changes since this repo was updated
I will fix it

**Comment by jburel** [comment]
Looking the various files
Do we want support for both ``http`` and ``https``?

File: omero_marshal/__init__.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def get_schema_version(version):
         raise Exception("Unsupported OMERO version: " + version)
 
 SCHEMA_VERSION = get_schema_version(omero_version)
-BASE_URL = 'http://www.openmicroscopy.org/Schemas'
+BASE_URL = 'https://www.openmicroscopy.org/Schemas'
```
Comment (chris-allan) [inline omero_marshal/__init__.py]: These are used as namespaces. I don't think this is a good idea.

**Comment by jburel** [comment]
Closing

**Pull request closed without merging**